src/App.js

```diff
@@ -13,6 +13,7 @@ class App extends Component {
     this.state = {
       movies: [],
       individualMovie: null,
```

Comment:
Assigning 'null' to 'individualMovie' might create some bugs in your code. The best practice is always to assign the data type that you're expecting to receive from the API. Is 'individualMovie' an object, array, number, or string?